src/lib/instrument-listen.js

```diff
@@ -33,14 +35,14 @@ module.exports = (app, meta, initPromises) => {
 		app.use(raven.errorHandler());
 
 		// Optional fallthrough error handler
-		app.use((err, req, res, next) => { // eslint-disable-line no-unused-vars
+		app.use(/** @type {} */(err, req, res, next) => { // eslint-disable-line no-unused-vars
```

Comment:
**question**: i wouldn't expect this callback to be cast to an empty object here, is there a reason for it?